demisto_sdk/commands/validate/validators/BA_validators/BA127_is_valid_context_path_depth.py

```diff
@@ -0,0 +1,116 @@
+            if isinstance(content_item, Script):
+                script_paths = self.create_outputs_set(content_item)
+                invalid_paths_str = self.is_context_depth_larger_than_five(script_paths)
+                if invalid_paths_str:
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message.format(
+                                "script", content_item.name, invalid_paths_str
+                            ),
+                            content_object=content_item,
+                        )
+                    )
+            else:
+                command_paths = self.create_command_outputs_dict(content_item)
+                invalid_paths_dict = (
+                    self.is_context_depth_larger_than_five_integration_commands(
+                        command_paths
+                    )
+                )
+                if invalid_paths_dict:
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message.format(
+                                "command",
+                                invalid_paths_dict["command"],
+                                invalid_paths_dict["wrong_paths"],
+                            ),
+                            content_object=content_item,
+                        )
+                    )
+        return results
+
+    def is_context_depth_larger_than_five_integration_commands(
+        self, command_paths: dict
+    ) -> dict:
+        """Validate that all outputs entry has contextPath key for a given command.
+
+        Args:
+            command (Command): The command to run on
+
+        Returns:
+           List of bad context paths if the contextPath depths is bigger then 5. Otherwise, return False.
+        """
+        message = {}
+        for command_name, command_outputs in command_paths.items():
+            if wrong_values_string := self.is_context_depth_larger_than_five(
+                command_outputs
+            ):
+                message["command"] = command_name
+                message["wrong_paths"] = wrong_values_string
+        return message
+
+    def is_context_depth_larger_than_five(self, outputs: Set[str]) -> str:
+        """Validate that all outputs entry has contextPath key for a given command.
+
+        Args:
+            content_item (Iterable[ContentTypes]: The content item to run on
+
+        Returns:
+             List of bad context paths if the contextPath depths is bigger then 5. Otherwise, return False.
+        """
+        wrong_values_string = ""
+        wrong_depth_values = [
+            output for output in outputs if len(output.split(".")) > 5
+        ]
+        if wrong_depth_values:
+            wrong_values_string = "\n".join(wrong_depth_values)
+        return wrong_values_string
```

Comment:
```suggestion
        return "\n".join(wrong_depth_values)
```

Reply:
in the main function the return is going to invlalid_path_str:
```
invalid_paths_str = self.is_context_depth_larger_than_five(script_paths)
                if invalid_paths_str:
```
if I return a string with "\n" even if wrong_depth_values is empty, the validaiton will fail - since there will be result.

@YuvHayun are yu sure this is a good idea?